apps/meteor/lib/isOTRMessage.ts

```diff
@@ -0,0 +1,3 @@
+import { IMessage } from '@rocket.chat/core-typings';
+
+export const isOTRMessage = (message: IMessage): boolean => message.t === 'otr' || message.t === 'otr-ack';
```

Comment:
what do you about to move `isOTRMessage` and  `isEncryptedMessage` to core types?